src/commands/openYAMLConfigFile.ts

```diff
@@ -45,14 +47,67 @@ export async function openYAMLConfigFile(context: IActionContext, node?: StaticW
     await window.showTextDocument(configDocument, { selection });
 }
 
-async function getSelection(configDocument: TextDocument, buildConfigToSelect: BuildConfig): Promise<Range | undefined> {
-    const configRegex: RegExp = new RegExp(`${buildConfigToSelect}:`);
+/* eslint-disable @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access, no-prototype-builtins, @typescript-eslint/no-unsafe-assignment */
+export async function getSelection(configDocument: TextDocument, buildConfigToSelect: BuildConfig): Promise<Range | undefined> {
```

Comment:
Since the code is a bit complex and has some sketchy "any" types, I'd probably rename this to `tryGetSelection` and add a big try/catch that just returns `undefined` if an error happens. This function already allows `| undefined` in the return type anyways